api/src/main/java/org/pmiops/workbench/api/AppsController.java

```diff
@@ -2,15 +2,12 @@
 
 import javax.inject.Provider;
 import org.apache.arrow.util.VisibleForTesting;
+import org.pmiops.workbench.config.WorkbenchConfig;
 import org.pmiops.workbench.db.model.DbUser;
 import org.pmiops.workbench.db.model.DbWorkspace;
 import org.pmiops.workbench.leonardo.LeonardoApiClient;
 import org.pmiops.workbench.leonardo.LeonardoApiHelper;
-import org.pmiops.workbench.model.CreateAppRequest;
-import org.pmiops.workbench.model.EmptyResponse;
-import org.pmiops.workbench.model.ListAppsResponse;
-import org.pmiops.workbench.model.UserAppEnvironment;
-import org.pmiops.workbench.model.WorkspaceAccessLevel;
+import org.pmiops.workbench.model.*;
```

Comment:
I've updated my IntelliJ to avoid this. The official instructions for this are interesting:

<img width="690" alt="image" src="https://user-images.githubusercontent.com/31020403/222240817-f1cf6931-db81-40ab-a8e1-ae2aa3498198.png">

Reply:
That is also my maximum tab count